### DISCUSSION TITLE
Please make font-size configurable

### DISCUSSION THREAD

**Discussion by gimplyworxs** [keepassxreboot/keepassxc#9692] [Ideas]
I just tried out KeePassXC 2.7.5
Compared to previous versions that I used, the font size is so small now.
Can't it be as it was in previous versions, or at least configurable?
It's now nearly unusable for my old eyes.

**Comment by droidmonkey** [comment]
We have this on the docket. If you font size is too small then you likely have environment variables defined such as QT_SCALE_FACTOR.

**Comment by gimplyworxs** [comment]
The last usable version for me was 2.7.1.
I wonder why everybody thinks that tiny fonts look more modern.
It doesn't but makes life for us older people harder.

**Reply by droidmonkey** [reply]
We have not changed the font size at all

**Comment by gimplyworxs** [comment]
I've made screenshots of both 2.7.1 and 2.7.5.
Both the same system, the same file. 
2.7.1:
![kp_2 7 1](https://github.com/keepassxreboot/keepassxc/assets/59622701/b7f47b66-c0d3-4e52-87ca-c212405c4670)

2.7.5:
![kp_2 7 5](https://github.com/keepassxreboot/keepassxc/assets/59622701/7000b461-36e2-4c57-8882-fef6b2cb09ad)

There is a difference. So something must have changed.

**Reply by droidmonkey** [reply]
Not on our end... you need to check your environment variables.

**Comment by gimplyworxs** [comment]
Do they change just by opening one appimage, closing it and opening the other appimage?

**Reply by droidmonkey** [reply]
Hard to tell, appimage could be picking up an environment variable it ignored in the past. Not an application issue though. We do recommend using flatpak for a more uniform experience.

**Comment by esledov** [comment]
Just wanted to add, since I had the same problem after switching to a high-resolution monitor. You need to add something like "export QT_SCALE_FACTOR=2" to ~/.xprofile, setting it in /etc/environment doesn't seem to work.
